docker-for-mac/release-notes.md

```diff
@@ -13,35 +13,65 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
-## Docker Desktop 3.3.3
-2021-05-06
+## Docker Desktop 3.4.0
+2021-06-07
 
 [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
 [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
 
-## Upgrades
+### New
+### Upgrades
+
+- [Docker Engine 20.10.7](https://docs.docker.com/engine/release-notes/#20107)
+- [Compose CLI v1.0.16](https://github.com/docker/compose-cli/releases/tag/v1.0.16)
+- [Kubernetes 1.21.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.1)
+- [containerd v1.4.6](https://github.com/containerd/containerd/releases/tag/v1.4.6)
+- [runc v1.0.0-rc95](https://github.com/opencontainers/runc/releases/tag/v1.0.0-rc95)
+- [Go 1.16.5](https://github.com/golang/go/releases/tag/go1.16.5)
+
+### Bug fixes and minor changes
+
+- Prevent `docker run` from hanging if inotify event injection fails. Fixes [docker/for-mac#5590](https://github.com/docker/for-mac/issues/5590).
+- Fixed error showing stderr log in the UI. Fixes [docker/for-mac#5688](https://github.com/docker/for-mac/issues/5688).
+- Fix emulation of `riscv64`. Fixes [docker/for-mac#5699](https://github.com/docker/for-mac/issues/5699).
+- Automatically reclaim space after deleting containers by deleting volumes and removing build cache.
+- Respect the bypass setting in the manual proxy configuration when pulling and pushing images. See [docker/for-mac#2467](https://github.com/docker/for-mac/issues/2467).
+- Fixed Docker Compose application removal from the UI which with custom file name other than `docker-compose.yml`. Fixes [docker/for-win#11046](https://github.com/docker/for-win/issues/11046)
```

Comment:
How about " Fixed an issue which caused Docker Compose application to be removed from Docker Desktop if it contains a file name other than `docker-compose.yml`."

Reply:
It’s actually the other way around I believe: the application couldn’t be removed.

Reply:
Aha but indeed that sentence doesn’t make any sense 😀 
How about «Fixed deleting Docker Compose applications not removing from the UI when based on a file name different from `docker-compose.yml`.»?

Reply:
Aah OK. Thanks for clarifying. I thought the application was automatically removed if it contained a custom file name :D 

Flipped the order to see if this reads better. "Docker Compose applications with file names other than `docker-compose.yml` can now be removed from Docker Desktop."

Reply:
Yes, that’s better. Thanks!